src/translations/components/TranslationsAttributesPage/TranslationsAttributesPage.tsx

```diff
@@ -110,64 +112,50 @@ const TranslationsProductTypesPage: React.FC<TranslationsProductTypesPageProps>
         onSubmit={onSubmit}
       />
       <CardSpacer />
-      <TranslationFields
-        activeField={activeField}
-        disabled={disabled}
-        initialState={true}
-        title={intl.formatMessage(messages.values)}
-        fields={
-          data?.attribute?.choices?.edges?.map(
-            ({ node: attributeValue }, attributeValueIndex) => {
-              const isRichText =
-                attributeValue?.inputType === AttributeInputTypeEnum.RICH_TEXT;
-              const displayName = isRichText
-                ? intl.formatMessage({
-                    defaultMessage: "Text",
-                    description: "attribute richtext value"
-                  })
-                : intl.formatMessage(
-                    {
-                      defaultMessage: "Value {number}",
-                      description: "attribute values"
-                    },
-                    {
-                      number: attributeValueIndex + 1
-                    }
-                  );
+      {data && withChoices && (
```

Comment:
```suggestion
      {withChoices && (
```
`data` is already checked within `withChoices`

Reply:
Changed